libs/shared/selectors/src/selectors/boolean-option-selector/boolean-option-selector.component.html

```diff
@@ -0,0 +1,4 @@
+<select [formControl]="control()" [id]="controlName()" class="form-control form-control-sm">
```

Comment:
what do we need `controlName()` for? I think it can be avoided (and the `id` be set in the parent component)

Reply:
> it's to be coherent with the structure before, so from the id you can access the formControl

I think it is better to move it out, because we do not need it. If you want to access the formControl, you can still do by first accessing the outer component and then querying inside it